xpk.py

```diff
@@ -1212,13 +1262,14 @@ def run_gke_node_pool_create_command(args, system_characteristics) -> int:
         f' --region={zone_to_region(args.zone)}'
         f' --num-nodes={system_characteristics.vms_per_slice}'
         f' --machine-type={system_characteristics.gce_machine_type}'
-        f' --tpu-topology={system_characteristics.topology}'
         f' --host-maintenance-interval={args.host_maintenance_interval}'
         f' {capacity_args}'
         ' --scopes=storage-full,gke-default'
         ' --enable-gvnic --max-pods-per-node 15'
         f' {args.custom_tpu_nodepool_arguments}'
     )
+    if args.tpu_type:
```

Comment:
Use if accelerator type here

Reply:
we don't have args.accelerator_type. we have `tpu_type=None` and `gpu_type=xxx` if accelerator is gpu, and vice versa and thus using `if args.tup_type` to check the accelerator

Reply:
outdat4ed